render/detailed/connections.go

```diff
@@ -29,13 +30,42 @@ var (
 	}
 )
 
+// ConnectionsTable is the table of connection to/form a node
+type ConnectionsTable struct {
```

Comment:
Having a separate type for these is good. Not convinced on the naming, as I'd like to move away from the Table/Row nomenclature (only used by Metadata/MetricRows anymore).